sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -689,8 +689,9 @@ def examples_wordcount_streaming(argv):
 
     output = (
         lines
-        | 'DecodeUnicode' >>
-        beam.FlatMap(lambda encoded: encoded.decode('utf-8'))
+        | 'DecodeUnicode' >> beam.FlatMap(
+            lambda encoded:
+            (encoded if isinstance(encoded, list) else encoded.decode('utf-8')))
```

Comment:
We [changed](https://github.com/apache/beam/pull/12352/files#diff-2e9994999827107bfae342b995425f08R782) a test to use a list instead of bytes, since it was written incorrectly

Reply:
Oh yeah I see. The test is wrong now, but it was also wrong before because it returned an output value of type `bytes`, which we now throw an error for in `process_outputs` of `common.py`. We had an email thread about this earlier which I'll send a reply to for more context.

Reply:
This should be changed to `beam.Map(lambda encoded: encoded.decode('utf-8'))` and the test values below changed back to bytes.